README.md

```diff
@@ -153,6 +153,9 @@ and apps that depend on each other will be executed in the correct order.
 We support several test frameworks as first-class citizens.
 That's a testing tool after all!
 
+Note that the Django `post_migrate` signal's receiver list is cleared at
+the start of tests and restored afterwards. If you need to test your
+own `post_migrate` signals then attach/remove them during a test.
```

Comment:
Just a small nitpick: can you please add an empty line between text and `### pytest` header?